### DISCUSSION TITLE
Allow me to select 24h/am-pm time displays

### DISCUSSION THREAD

**Discussion by toothbrush** [plausible/analytics#217] [Anything Else]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the feature has not yet been requested.

## Feature request

This is quite minor, but i would like the option to choose 24-hour time reporting, instead of am/pm.  

![image](https://user-images.githubusercontent.com/423357/87237172-5a457180-c436-11ea-885d-34c6535e26ce.png)

Looking at the `?period=day` chart, it's saying `9pm` (also all along the horizontal axis), whereas for me it's more natural to say `21:00`.

**Update:** The time format (12 hour or 24 hour) for top graph timelines is now presented automatically based on the browser's defined language.

**Comment by maique** [comment]
+1

**Comment by smiklosovic** [comment]
+1

**Comment by oldrup** [comment]
Developed and hosted in Europe. Surely time and dates can be viewed in the preferred language set in the browser, or?

![billede](https://user-images.githubusercontent.com/17141205/175876506-88a68a10-5d34-41ef-a1ec-1d416387cbc0.png)

This is a major UX drawback imo - and surely an oversight, I suppose. Please follow the date format of the user.

**Comment by metmarkosaric** [comment]
**Update:** The time format (12 hour or 24 hour) for top graph timelines is now presented automatically based on the browser's defined language.

**Reply by oldrup** [reply]
Perfect! Thank you! 🙏

**Reply by JulyIghor** [reply]
> **Update:** The time format (12 hour or 24 hour) for top graph timelines is now presented automatically based on the browser's defined language.

Why make it depend on a language? How do you force 24-hour format in English?
